### PR TITLE
CLDR-17843 Update xnr.xml -- Add scaffolding for Kangri Devanagari nu…

### PR DESCRIPTION
**Pull request by conradarcturus**
…mber formatting patterns

We don't have submitted data for the Devanagari numerals in the Kangri language -- so we'll fall back to what Hindi is doing, which is just to inherit parent values atm.

This will prevent Kangri from dropping in coverage level.

CLDR-17843

- [x] This PR completes 3/7 locales mentioned in the ticket.

ALLOW_MANY_COMMITS=true
DISABLE_JIRA_ISSUE_MATCH=true

### CODE REVIEW

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
remember we'll need to regenerate coverageLevels.txt after this merges as in #3897

File: common/main/kxv.xml [hunk 1]
```diff
@@ -3408,6 +3408,20 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
+				</decimalFormat>
+			</decimalFormatLength>
+		</decimalFormats>
+		<scientificFormats numberSystem="latn">
+			<scientificFormatLength>
+				<scientificFormat>
+					<pattern>↑↑↑</pattern>
+				</scientificFormat>
+			</scientificFormatLength>
+		</scientificFormats>
```
Comment (srl295) [inline common/main/kxv.xml:3424]:
```suggestion
```

I think this is a duplicate of the following element

**Review by srl295 — DISMISSED** [review]
needs a fix

**Comment by conradarcturus** [comment]
Fix applied, please re-review

**Review by srl295 — APPROVED** [review]

**Pull request merged**